Packages/com.unity.inputsystem/InputSystem/Editor/Internal/AdvancedDropdown/AdvancedDropdownWindow.cs

```diff
@@ -518,7 +518,14 @@ protected void GoToParent()
             else
                 m_NewAnimTarget = -1;
             m_AnimationTree = m_CurrentlyRenderedTree;
-            m_CurrentlyRenderedTree = m_ViewsStack.Pop();
+            var parentItem = m_ViewsStack.Pop();
+
+            m_State.ClearSelectionOnItem(m_CurrentlyRenderedTree);
+
+            var suggestedIndex = parentItem.children.IndexOf(m_CurrentlyRenderedTree);
```

Comment:
Can parentItem be null here?

Reply:
I don't think so, as it always has a root and we can't go higher than a root item. But I can add a check just to be extra sure :)

Reply:
Fixed